architecture-decision-records/0024-continuous-integration-service.md

```diff
@@ -0,0 +1,44 @@
+# 24. Continuous integration service
+
+**Date:** 2022-03-15
+
+## Context
+We need a continuous integration service. There are two main functions we need from the service.
+* Test code changes before they are merged and deployed.
+* Build our code and deploy it to the various different services and environments it runs in.
+
+## Options considered
+We have been using Jenkins for a couple of years now so one option was to carry on using that.
+
+The other option we considered is using [GitHub actions](https://docs.github.com/en/actions).
+
+## Decision
+We have decided to move from Jenkins to GitHub actions. There are advantages and disadvantages to this move.
+
+### Advantages
+
+#### Server maintenance
+Jenkins runs on an EC2 instance using multiple docker images and uses multiple plugins and all of these need to be kept up to date separately which takes up quite a lot of time. 
```

Comment:
Reword slightly: _... and uses multiple plugins. These need to be kept up to date ..._